canonicalwebteam/blog_extension/app.py

```diff
@@ -0,0 +1,12 @@
+from webapp.blog_ext.views import build_blueprint
```

Comment:
Did you leave this in by accident? I'm not sure calling a function outside this module, that you expect to exist inside the wrapping webapp is a good idea.

(also not crazy about a blueprint builder - seems kinda obfuscating)

Reply:
yes this was by testing on s.io I need to remove it